regression/snapshot-harness/nondet_initialize_static_arrays/main.c

```diff
@@ -0,0 +1,51 @@
+#include <assert.h>
+#include <stdlib.h>
+
+typedef struct st1
+{
+  struct st2 *to_st2;
+  int data;
+} st1_t;
+
+typedef struct st2
+{
+  struct st1 *to_st1;
+  int data;
+} st2_t;
+
+typedef struct st3
+{
+  st1_t *array[3];
+} st3_t;
+
+st1_t dummy1;
+st2_t dummy2;
+
```

Comment:
Hmm. The dummys are not needed in this test? Remove anything extraneous or we'll be wondering if we've overlooked something.

Reply:
Removed.